HW_1/task_4.py

```diff
@@ -0,0 +1,11 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+#  Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+#  а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+
+s = int(input())
+print((s//6), ((s//6)*4), (s//6))

```

Comment:
Неплохо бы добавить проверку входных данных на кратность 6. Кроме того у операций деления и умножения одинаковый приоритет, значит, скобки не нужны. В-третьих у вас повторяется одно и то же действие, а именно деление на 6. Для этого используют переменные.
```suggestion
print( var := s // 6, var * 4, var)

```